fludashboard/libs/views.py

```diff
@@ -180,7 +180,7 @@ def data__weekly_incidence_curve(
         pass
 
     # cheating: using a new field corredor_muito_alto just for plotting
-    df['typical_very_high'] = df.very_high_threshold.max() * 1.02
```

Comment:
The additional columns will guarantee that it is always above any other curve in the chart:

    df['typical_very_high'] = df[['very_high_threshold', 'ci_upper', 'value', 'typical_high']].max().max() * 1.02

Reply:
done :)